ui/src/app/job-details/common/debug-icons/debug-icons.component.ts

```diff
@@ -97,6 +103,25 @@ export class JobDebugIconsComponent implements OnInit {
     }
   }
 
+  // If this is a GCP Batch operation, transform the operation id into a URL to the Batch 
+  // job details page.
+  // Example input: projects/broad-dsde-cromwell-dev/locations/us-central1/jobs/job-1a4f7cff-3f17-49bf-b9ef-48b8b09c0f39
+  // Example output: https://console.cloud.google.com/batch/jobsDetail/regions/us-central1/jobs/job-1a4f7cff-3f17-49bf-b9ef-48b8b09c0f39/details?project=broad-dsde-cromwell-dev
+  getOperationDetailsUrl(): string {
+    var match = this.gcpBatchOperationIdRegex.exec(this.operationId);
+    if (match != null) {
+      var projectId = match.groups.projectId;
+      var location = match.groups.location;
+      var batchJobId = match.groups.batchJobId;
+      return `https://console.cloud.google.com/batch/jobsDetail/regions/${location}/jobs/job-${batchJobId}/details?project=${projectId}`;
+    }
+    else {
+      return '';
+    }
+  }
+
+  // If this task ran on the Cromwell GCP Batch backend, link directly to the GCP Console for
+  // operation details. If it ran on the older PAPIv2, load the operation details into a modal.
```

Comment:
I feel like `hasExternalOperationDetails` is a better place for this comment, as it actually contains the conditional

Reply:
Good call, neglected to update this comment after a small refactor.